release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -4583,4 +4583,25 @@ $ oc adm release info 4.13.59 --pullspecs
 
 [id="ocp-4-13-59-updating_{context}"]
 ==== Updating
+To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+// 4.13.60
+[id="ocp-4-13-60_{context}"]
+=== RHBA-2025:15675 - {product-title} 4.13.60 bug fix and security updates
+
+Issued: 18 September 2025
+
+{product-title} release 4.13.60, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2025:15675[RHBA-2025:15675] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:24218[RHBA-2025:24218] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
```

Comment:
You can remove this sentence because there are no bugs to be documented in this release.